data/publiccloud/terraform/azure.tf

```diff
@@ -88,6 +88,11 @@ variable "ssh_public_key" {
   default = "/root/.ssh/id_rsa.pub"
 }
 
+locals {
```

Comment:
we already have `subnet_id` why do we need this ?

Reply:
If `var.subnet_id`, means the variable `subnet_id` being handed over via the command line is not set, this is a container for the new value.

`var.subnet_id` should be immutable as far as I know, thus conditional changes without having another variable are impossible.